plugins/knife/knife-deployment-resize.rb

```diff
@@ -118,18 +118,17 @@ class DeploymentResize < Chef::Knife
 
     def get_current_counts
       result = Hash.new(0)
-      %w{bono ellis ibcf homer homestead sprout sipp ralf seagull}.each do |node|
+      %w{bono ellis ibcf homer homestead sprout sipp ralf seagull database ralfstead}.each do |node|
         result[node.to_sym] = find_nodes(roles: "chef-base", role: node).length
       end
       return result
     end
 
     def update_ralf_hostname environment, cloud
       ralfs = find_nodes(roles: "chef-base", role: "ralf").length
-
       changed_nodes = []
 
-      %w{bono ibcf sprout ralf}.each do |node_type|
+      %w{bono ibcf sprout ralf ralfstead}.each do |node_type|
```

Comment:
I think this tails over from when i was trying to plumb the ralf attribute through in split-storage land. 
I'm going to make it just skip the setting has_ralf step if split_storage. Sound reasonable?